megatron/core/models/gpt/linear_attention_module_specs.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+
+from typing import Optional
+
+from megatron.core.models.backends import BackendSpecProvider
+from megatron.core.ssm.gated_delta_net import GatedDeltaNet, GatedDeltaNetSubmodules
+from megatron.core.ssm.mamba_mixer import MambaMixer, MambaMixerSubmodules
+from megatron.core.transformer.spec_utils import ModuleSpec
+
+
+def get_linear_attention_module_spec_for_backend(
+    backend: BackendSpecProvider, linear_attention_type: str, normalization: Optional[str] = None
+) -> ModuleSpec:
+    """Helper function to get module spec for Linear Attention"""
+    rms_norm = normalization == "RMSNorm"
+    if linear_attention_type == "mamba":
```

Comment:
Why can GPT models now have mamba attention? So what's the difference now between a GPT model and a Mamba Model? GPT Model is just more confusing?